debian/pt-web-portal.install

```diff
@@ -1,7 +1,5 @@
-pkg/pwdptwp.sh                              /usr/lib/pt-web-portal
-src/expand-fs.sh                            /usr/lib/pt-web-portal
-src/notify-on-port-busy.sh                  /usr/lib/pt-web-portal
-src/server/Pipfile*                         /usr/lib/pt-web-portal
-src/server/run.py                           /usr/lib/pt-web-portal
-src/server/backend                          /usr/lib/pt-web-portal
+debian/extras/notify-on-port-busy.sh        /usr/lib/pt-web-portal
+debian/extras/pwdptwp.sh                    /usr/lib/pt-web-portal
 src/miniscreen                              /usr/lib/pt-web-portal
+src/server/backend                          /usr/lib/pt-web-portal
+src/server/run.py                           /usr/lib/pt-web-portal
```

Comment:
If we get the entrypoint handled via a Python build stage (see `pi-top` command in SDK, or `pt-miniscreen` in the Miniscreen App's `hypermodern` branch for an example), we can simply say to install any scripts that are installed into `/usr/bin` with this:
```suggestion
usr/bin
```